cypress/e2e/gui.cy.js

```diff
@@ -0,0 +1,145 @@
+/// <reference types = "cypress" />
+
+describe('EngageSphere gui test cases', () => {
+beforeEach(() => {
+  Cypress.on('window:before:load', window => {
+    window.document.cookie = 'cookieConsent=accepted'
+  })
+  cy.visit('/')
+})
+
+    it('Keeps the filters when coming back from the customer details view ', () => {
+      cy.get('select[data-testid="size-filter"]')
+       .should('be.visible')
+       .select('Medium')
+      cy.contains('button', "View")
+       .click()
+      cy.contains('button', "Back")
+       .click()
+      cy.get('select[data-testid="size-filter"]')
+       .should('have.value', 'Medium')
+    })
+    it('It goes back to the customers list when clicking the "Back" button *', () => {
+      cy.contains('button', "View")
+       .click()
+      cy.contains('button', "Back")	
+       .click()
+      cy.contains('p', "Below is our customer list.")
+       .should('be.visible')
+    })
+
+    it('It renders the footer with the right text and links *', () => {
+      cy.get('footer')
+       .should('be.visible')
+       .contains('Copyright 2024 - Talking About Testing')
+      cy.get('a[href="https://hotmart.com/pt-br/club/cypress-playground-ate-a-nuvem"]')
+       .should('have.text', 'Hotmart')
+      cy.get('a[href="https://udemy.com/user/walmyr" ]')
+       .should('have.text', 'Udemy')
+      cy.get('a[href="https://talkingabouttesting.com" ]')
+       .should('have.text', 'Blog')
+      cy.get('a[href="https://youtube.com/@talkingabouttesting" ]')
+       .should('have.text', 'YouTube')
+    })
+
+    it('It renders the "Hi there" greeting when no name is provided *', () => {
+      cy.contains('h2', 'Hi there')
+       .should('be.visible')
+    })
+
+    it('It renders the "Hi Joe" greeting when name is provided *', () => {
+      cy.get('input[id=name]')
+       .type('Joe')
+       cy.contains('h2', 'Hi Joe')
+       .should('be.visible')
+    })
+
+    it('It renders the header with a heading, theme`s toggle, and a text input field', () => {
+      cy.get('h1')
```

Comment:
Em vez de:

```js
cy.get('h1')
  .should('be.visible')
  .contains('EngageSphere')
```

Poderia ser: `cy.contains('h1', 'EngageSphere').should('be.visible')`.

Mais simples, não?

Reply:
Algum motivo para não implementar esta sugestão?